src/CFamily.UnitTests/packages.lock.json

```diff
@@ -1019,8 +1019,8 @@
       },
       "Newtonsoft.Json": {
         "type": "Transitive",
-        "resolved": "13.0.1",
-        "contentHash": "ppPFpBcvxdsfUonNcvITKqLl3bqxWbDCZIzDWHzjpdAHRFfZe0Dw9HmA0+za13IdyrgJwpkDTDA9fHaxOrt20A=="
+        "resolved": "13.0.3",
```

Comment:
Most of the changes are updates to `package.lock.json`. I'll add review comments for all of the non-package lock changes.

I've checked the `Integration.Vsix.UnitTests` package lock file has been updated correctly i.e. it doesn't have the extra "GraphModel" version changes that cause problems.